src/libsyntax_pos/hygiene.rs

```diff
@@ -508,7 +529,7 @@ impl SyntaxContext {
     pub fn reverse_glob_adjust(&mut self, expn_id: ExpnId, glob_span: Span)
                                -> Option<Option<ExpnId>> {
         HygieneData::with(|data| {
-            if data.adjust(self, expn_id).is_some() {
+            if data.adjust(&mut {*self}, expn_id).is_some() {
```

Comment:
Could you explain what happens here?

Reply:
`adjust` will undo `SyntaxContext::unique` here, so it's now passed a copy because we don't want that to happen to `*self`. I'll change this to be more clear if this remains the exact approach used.